aci/data_source_aci_fvrsprovcons.go

```diff
@@ -17,34 +17,33 @@ func dataSourceAciContractProvider() *schema.Resource {
 
 		SchemaVersion: 1,
 
-		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+		Schema: AppendAttrSchemas(map[string]*schema.Schema{
```

Comment:
Why is this change being made here if GetAnnotationAttrSchema() is not added at the bottom?

Reply:
description - is not allowed for the Contract object objects, So to ignore the description field I removed AppendBaseAttrSchema.